### ISSUE TITLE
[NPC] Doomwalker - evade

### ISSUE THREAD

**Issue by Godmatik** [Atlantiss/NetherwingBugtracker#8339]
**Current behaviour**:
Doomwalker will occasionally evade mid-combat and completely reset.

**Expected behaviour**: 
He should not evade. 

We suspect it is happening due to the obstacles on the ground, he is targeting during earthquake who is on top of an obstacle, and there is no path, so he resets. Just a guess though.

**Server Revision**: 
3745

**Comment by Foodlenz** [comment]
I think this might be in relation to the latest pathing changes. I noticed a couple more with other mobs while doing the coilfang heroics last night that never used to happen.

**SV: Second pull (patrolling melee guy and caster).** 
Usually pull these two behind the dirt wall/pillar at the bottom of the steps. This now evades/resets them when their aggro target is in this position.

**SP: Quagmarian**
First pull i had him still in the water, while i was on land. He eventually evaded and reset.
Second pull I went into the water to tank him, but my white attacks weren't happening (might be a separate bug) could still yellow attack and hit him though. One of the ranged dps still on land pulled aggro and the boss reset. 
Third pull I dragged him completely out of the water and could white attack him and kill him.
